src/ngMessageFormat/messageformat.js

```diff
@@ -0,0 +1,938 @@
+'use strict';
+
+var angular = window['angular'];
+
+var $$messageFormatMinErr = angular['$$minErr']('$$messageFormat');
+
+// NOTE: ADVANCED_OPTIMIZATIONS mode.
+//
+// This file is compiled with Closure compiler's ADVANCED_OPTIMIZATIONS flag! Be wary of using
+// constructs incompatible with that mode.
+
+/**
+ * @ngdoc service
+ * @name $$messageFormat
+ *
+ * @description
+ * Angular internal service to recognize MessageFormat extensions in interpolation expressions.
+ * For more information, see:
+ * https://docs.google.com/a/google.com/document/d/1pbtW2yvtmFBikfRrJd8VAsabiFkKezmYZ_PbgdjQOVU/edit
+ */
+
+var noop = angular['noop'],
+    isFunction = angular['isFunction'],
+    toJson = angular['toJson'];
```

Comment:
Why use lookup (`angular['noop']`) rather than property (`angular.noop`) syntax here. Is it because of the Google Closure compilation optimizations?